app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ReservationModel.java

```diff
@@ -0,0 +1,57 @@
+package com.mobile.tandil.javabaseproject.mvp.model;
+
+import com.mobile.tandil.javabaseproject.database.ParkingDatabase;
+import com.mobile.tandil.javabaseproject.entity.Reservation;
+import com.mobile.tandil.javabaseproject.enums.ValidationEnums;
+import com.mobile.tandil.javabaseproject.mvp.contract.ReservationContract;
+import java.util.ArrayList;
+import java.util.Calendar;
+
+public class ReservationModel implements ReservationContract.Model {
+
+    ParkingDatabase database;
+
+    public ReservationModel(ParkingDatabase database) {
+        this.database = database;
+    }
+
+    @Override
+    public Boolean createReservation(Calendar startingDate, Calendar endingDate, String reservationCode, Integer parkingNumber) {
+        try {
```

Comment:
why do you use a try-catch here?  how would this throw an exception?

Reply:
It shouldn't, because im doing the validations before. Fixed

Reply:
to trigger the try-catch, the function you're executing should throw an exception, and in this case, it doesn't.

Reply:
Great. Fixed